jupyter-notebooks/analytics-snippets/README.md

```diff
@@ -0,0 +1,7 @@
+# Analytics Snippets
+
+This directory holds notebooks that demonstrate quick code snippets useful
+for working with the analytics output.
+
+Snippets include:
+- converting the roads raster to a vector dataset
```

Comment:
small detail: can we include the building footprints snippet as a bullet here too?

Reply:
ahh, great catch! on it